apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -0,0 +1,563 @@
+            return html` <option value="${user.user_id}">${user.uid}</option> `;
+          })}
+        </select>
+      </div>
+      <input type="hidden" name="__action" value="delete_member" />
+      <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+      <input type="hidden" name="group_id" value="${row.group_id}" />
+      <button
+        type="button"
+        class="btn btn-secondary"
+        onclick="$('#row${row.group_id}Popoverdeletemember').popover('hide')"
+      >
+        Cancel
+      </button>
+      <button type="submit" class="btn btn-danger" ${row.users.length > 0 ? '' : 'disabled'}>
+        Delete
+      </button>
+    </form>
+  `;
+}
+
+function UploadAssessmentGroupsModal({ csrfToken }: { csrfToken: string }) {
+  return Modal({
+    id: 'uploadAssessmentGroupsModal',
+    title: 'Upload new group assignments',
+    body: html`
+      <p>Upload a CSV file in the format of:</p>
+      <code class="mb-0 text-dark">
+        groupName,UID<br />
+        groupA,one@example.com<br />
+        groupA,two@example.com<br />
+        groupB,three@example.com<br />
+        groupB,four@example.com</code
+      >
+      <!-- Closing code tag not on its own line to improve copy/paste formatting -->
+      </div>
+      <div class="modal-body">
+        <p>
+          The <code>groupname</code> column should be a unique identifier for each group. To
+          change the grouping, link uids to the groupname.
+        </p>
+        <form name="upload-assessment-group-form" method="POST" enctype="multipart/form-data">
+          <div class="form-group">
+            <div class="custom-file">
+              <input
+                type="file"
+                accept=".csv"
+                name="file"
+                class="custom-file-input"
+                id="uploadAssessmentGroupsFileInput"
+              />
+              <label class="custom-file-label" for="uploadAssessmentGroupsFileInput">
+                Choose CSV file
+              </label>
+            </div>
+          </div>
+          <div class="d-flex justify-content-end">
+            <div class="form-group mb-0">
+              <input type="hidden" name="__action" value="upload_assessment_groups" />
+              <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+              <button type="button" class="btn btn-secondary" data-dismiss="modal">
+                Cancel
+              </button>
+              <button type="submit" class="btn btn-primary">Upload</button>
+            </div>
+          </div>
+        </form>
+      </div>
+    `,
+  });
+}
+
+function AutoAssessmentGroupsModal({
+  groupMin,
+  groupMax,
+  csrfToken,
+}: {
+  groupMin: number;
+  groupMax: number;
+  csrfToken: string;
+}) {
+  return Modal({
+    id: 'autoAssessmentGroupsModal',
+    title: 'Auto new group setting',
+    body: html`
+      <form name="auto-assessment-group-form" method="POST" enctype="multipart/form-data">
```

Comment:
As with the other modals: drop this in favor of the default one provided by `Modal`.

Reply:
Now using the built-in form in [dc8a69d](https://github.com/PrairieLearn/PrairieLearn/pull/9550/commits/dc8a69d5bba27345198abea501037c2deb732050).